test/test_pipeline_schedule.py

```diff
@@ -3,11 +3,13 @@
 import unittest
 
 import torch
+import torch.distributed as dist
 import torch.nn as nn
-from pippy.PipelineSchedule import PipelineStageV2Impl
+from pippy.PipelineSchedule import dry_run, PipelineStageV2Impl, validate_stages
```

Comment:
nit: if `dry_run` is going to be a user-facing util, maybe give it a more distinguishing name? (because users can have their own `dry_run` functions referring to other things). In fact, it may be even worth considering `deviating` from `dry_run` (implementation detail) and just name it based on purpose (e.g. `get_stage_shapes`, etc)

Reply:
makes sense, renamed it